getl/blocks/load/entrypoint.py

```diff
@@ -52,28 +52,30 @@ def batch_json(conf: BlockConfig) -> DataFrame:
     :param str FileRegistry=: name of the fileregistry to use
     :param str Alias=: an alias for the dataframe that is loaded
     :param str Suffix=.json: the suffix of the file
+    :param str JsonSchemaPath=: the file schema in json format
 
 
     ```
     SectionName:
         Type: load::batch_json
         Properties:
-            Path: s3://husvarna-datalake/trusted/amc/live
+            Path: s3://husqvarna-datalake/trusted/amc/live
```

Comment:
Wops, we should probably rename this to s3://bucket/path/to/files because its an open source repo.